client/landing/subscriptions/components/site-list/site-row.tsx

```diff
@@ -68,25 +69,36 @@ export default function SiteRow( {
 		SubscriptionManager.useSiteEmailMeNewCommentsMutation();
 	const { mutate: unsubscribe, isLoading: unsubscribing } =
 		SubscriptionManager.useSiteUnsubscribeMutation();
+
+	const navigate = useNavigate();
+
+	const individualPagePath = useMemo( () => {
```

Comment:
There is no need to `useMemo` here, in fact the cost of using is greater than the benefit. 
The following is a good article to give you an idea of when it is beneficial to use the state management hooks: https://kentcdodds.com/blog/usememo-and-usecallback.